app/slashcommands/command_custom_status.go

```diff
@@ -19,7 +20,8 @@ const (
 	CmdCustomStatus      = app.CmdCustomStatusTrigger
 	CmdCustomStatusClear = "clear"
 
-	DefaultCustomStatusEmoji = "speech_balloon"
+	DefaultCustomStatusEmoji    = "speech_balloon"
+	DefaultCustomStatusDuration = "today"
```

Comment:
Why is the default to expire after one day, and not simply stay forever?

Reply:
It was in the spec - https://mattermost.atlassian.net/wiki/spaces/CS/pages/983302547/User+Experience+and+Functional+Notes

Reply:
@esethna, gut feel is that we should default the user interface to expire after one day, but keep the expiration optional and defaulting to never if not explicitly specified in the REST API.

Reply:
@lieut-data I think that is the intention if I'm understanding correctly. The UI selector shows "today" as the default option when the users go set their status (and as such statuses will clear the next day if this is not changed), but users can change it to "Don't clear" if they want it to live forever.

![image](https://user-images.githubusercontent.com/14333569/117380147-f3de8380-ae8d-11eb-9de2-e5f8b5216cb1.png)

Reply:
Sounds good, @esethna! @manojmalik20, can we make that distinction, namely to have the UI default as described, but leave the default of the REST API to never expire?